packages/perseus/src/widgets/cs-program/cs-program.tsx

```diff
@@ -15,10 +14,42 @@ import Util from "../../util";
 import {isFileProtocol} from "../../util/mobile-native-utils";
 import {toAbsoluteUrl} from "../../util/url-utils";
 
-import type {WidgetExports} from "../../types";
+import type {PerseusScore, WidgetExports} from "../../types";
 
 const {updateQueryString} = Util;
 
+type Status = "correct" | "incorrect" | "incomplete";
+
+type UserInput = {
+    status: Status;
+    message: string | null;
+};
+
+type Setting = {
+    name: string;
+    value: any;
+};
+
+type Props = {
```

Comment:
EDIT: I got through the rest and see you've done this on other widgets. I'm guessing you had a reason why you didn't do it here here. I'll leave the comment for posterity. 

Could you work `PerseusCSProgramWidgetOptions` in here instead of defining the props all independently?  I realize that the TS compiler matches by type but using the widget options defined in `perseus-types.ts` helps to paint the picture of which options are part of the schema.

Reply:
No, it was just one of my early conversions and I hadn't picked up the right pattern yet. Fixed, good catch.